account-integrations/safe/src/SafeECDSAPlugin.sol

```diff
@@ -19,21 +19,25 @@ interface ISafe {
     ) external returns (bool success);
 }
 
+struct ECDSAOwnerStorage {
```

Comment:
I also copied this pattern from Kernel since they support recovery as well.  I don't think it's needed to be a struct.  I do think it's a little more readable than just making the mapping `mapping(address => address)`

Reply:
makes sense, cheers

Reply:
Consider https://docs.soliditylang.org/en/latest/types.html#user-defined-value-types ?